ledger-bridge.js

```diff
@@ -80,14 +83,14 @@ export default class LedgerBridge {
 
     async attemptMakeApp (replyAction) {
         try {
-            await this.makeApp({ openOnly: true });
-            await this.cleanUp();
+            await this.makeApp({ openOnly: true })
+            //await this.cleanUp()
```

Comment:
Should this still be commented out? I think you mentioned something about how this was interferring with the polling?

However, I think this was added in specifically to prevent some error case... in particular, I think that without cleanup there could be an error about how ledger is already connected when attempting to connect after calling `attemptMakeApp`

Reply:
I commented this out for the sake of debugging; `cleanUp` kills the `clearInterval` I'm using during cleanup, which will then also signal disconnection.

Reply:
@darkwing But this cleanup was needed, or else there would be errors on attempts to connect the device after attemptMakeApp is called. Also, cleanup doesn't call `clearInterval`. I think we have to leave these `cleanup` calls as is. Perhaps if you need this attemptMakeApp method, but without the cleanup calls, you could add a separate method for that?